### DISCUSSION TITLE
Problems on bplaced.net

### DISCUSSION THREAD

**Discussion by Aza1261** [prasathmani/tinyfilemanager#736] [Q&A]
Hello guys,
This question might be stupid and I apologize in advance for that case.
I am using tinyfilemanager on my free bplaced.net Account.
it works fine so far but two features don't work.
Rename and New file. Both should (judging from the demo) open a popup for input. That doesn't happen for me.
Is there an easy explanation for this?

**Comment by michael-milette** [comment]
Sounds like your web server does not have permission to create files. Can you edit and save, copy or move files?

**Reply by Aza1261** [reply]
Hallo Michael,
thank you for your fast answer.
Editing, saving, copying and moving jseems to work.
But the sizecalculation now throws and error :o 
Fatal error: Uncaught Error: Call to undefined function disk_free_space() in /users//www/index.php:2144 Stack trace: #0 {main} thrown in /users//www/index.php on line 2144

**Reply by Aza1261** [reply]
I just realized the search function doesn't seem to work aswell.
